bcdaworker/queueing/manager/que.go

```diff
@@ -113,37 +114,44 @@ func (q *queue) processJob(job *que.Job) error {
 		return nil
 	}
 
+	ctxFields := logrus.Fields{"job_id": jobArgs.ID, "aco_id": jobArgs.ACOID}
```

Comment:
When we call logrus's `WithFields` function, it returns an entry and takes a map of fields of the type `logrus.Fields`. This is where the log entry starts to "snowball" fields and pick up relevant data as the worker processes the job.

Reply:
we later use the `cms_id`, so `aco_id` might not be something that we necessarily need or want here.

Reply:
I would prefer to remove the `aco_id` to be honest. I can have a look through our reporting to see if that ID is used anywhere and if it's not, am I OK to remove?